src/client/components/lazy-chat/providers/crisp.js

```diff
@@ -12,14 +12,12 @@ export function onAccepted () {
   $crisp.push(['do', 'chat:open'])
 }
 
-/**
- * Hooks to execute after script has been loaded
- *
- */
-export function onLoaded () {}
+export function listenToOpener (cb) {
```

Comment:
rename to `onSessionLoaded`

keep `onLoaded` there too, it's just unused in this provider